do-server/src/index.mjs

```diff
@@ -70,6 +70,22 @@ class ServerTransaction {
 const allowedOrigin = 'http://localhost:5173';
 const KV_NAMESPACE = 'room_keys_2';
 
+const getJWT = (request) => {
+	const authHeader = request.headers.get('Authorization');
+	if (!authHeader || !authHeader.startsWith('Bearer')) {
+		return null;
+	}
+
+	return authHeader.substring(6).trim();
+};
+
+const verifyAuth = async (request) => {
```

Comment:
I think ultimately this function is what will be defined by the developer and imported the same way as `mutators.js`, but since the CLI/npm packaging is still a wip we can probably pick that up later.

Reply:
Yeah, I think a lot of this can be left until we are clear exactly how are the mutators.js and authHandler.js files are being copied and moved to the server